network-api/networkapi/wagtailpages/pagemodels/libraries/rcc/detail_page.py

```diff
@@ -72,36 +70,30 @@ class RCCDetailPage(BasePage):
 
     content_panels = wagtail_models.Page.content_panels + [
         image_handlers.FieldPanel("cover_image"),
-        # TODO: Reactivate once links are implemented
-        # edit_handlers.InlinePanel("rcc_links", heading="Article links"),
+        edit_handlers.InlinePanel("rcc_links", heading="Article links"),
         edit_handlers.FieldPanel("original_publication_date"),
         edit_handlers.FieldPanel("introduction"),
         edit_handlers.FieldPanel("overview"),
-        # TODO: Reactivate once links are implemented
-        # edit_handlers.InlinePanel("rcc_authors", heading="Authors", min_num=1),
+        edit_handlers.InlinePanel("rcc_authors", heading="Authors", min_num=1),
         edit_handlers.FieldPanel("contributors"),
-        # TODO: Reactivate once links are implemented
-        # edit_handlers.InlinePanel("related_content_types", heading="Content types"),
-        # edit_handlers.InlinePanel("related_curricular_areas", heading="Curricular areas"),
-        # edit_handlers.InlinePanel("related_topics", heading="Topics"),
+        edit_handlers.InlinePanel("related_content_types", heading="Content types"),
+        edit_handlers.InlinePanel("related_curricular_areas", heading="Curricular areas"),
+        edit_handlers.InlinePanel("related_topics", heading="Topics"),
```

Comment:
Related to the `rcc_` prefix: here we are not using the prefix, while these are also inline panels. I think this is preferrable. I don't think the prefix adds to the understanding.